### PR TITLE
Release notes 3.3.7

### PR DESCRIPTION
**Pull request by solemabrothers**

### CODE REVIEW

File: release-notes/ugandaemr-337.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# 3.3.7 - March 2022
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Change to April 2022

File: release-notes/ugandaemr-337.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+1. Martnernal and Child Health
+  
+   * Antenatal  Services for both Point of Care and Restrospective approach. 
+   * Maternity Services
+   * Postnatal Services
+   * Reports - (105 MCH Section, ANC 105, Marternity 105, 105 PNC )
+   
+2. Safe Male Circumcision
+
+    *Retrospective Data Entry for SMC Services
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Make it a single sentence

File: release-notes/ugandaemr-337.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    
+3. Create a generic approach to generating  and Sync FHIR Encounters for data exchange
+4. Added Missing Districts
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Name the districts

File: release-notes/ugandaemr-337.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    
+## Improvements
+ 
+ 1. Add Display Board for Logs
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Where is this

Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: How is this important to the end user

Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Which results, Viral Load?

File: release-notes/ugandaemr-337.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+ 2. Add SNOMED CT Mapping to Fhir Concept Source Table
+ 3. Remove dependency on guava that is flagged as a potential security issue
+ 4. Checking of results in order to also check for results entered from the encounter
+ 5. Ensure that orders with Viral Load results are completed.
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md:23]: When are the orders completed

File: release-notes/ugandaemr-337.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+ 6. Improve VL Non Suppressed form Validation Rules
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Which validation rules were improved and how

File: release-notes/ugandaemr-337.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+ 7. Update Transfer-In Regimen to match the list of regimens in current regimen
+ 8. Differentiate between a pharmacy visit and a facility visit in ART Access on Patient Dashboard
+ 
+
+## Bug Fixes 
+
+1. Fix failing retrospective of Ordering Viral Load
+2. Fix appointments widget with some appointments not showing.
+3. Make return visit mandatory, transfer out and location required when patient is transferred out
+
```
Comment (ssmusoke) [inline release-notes/ugandaemr-337.md]: Remove all multiple spaces between headings

File: release-notes/ugandaemr-337.md (hunk 6 above)
Comment (solemabrothers) [reply release-notes/ugandaemr-337.md]: @Mwanje for your attention

File: release-notes/ugandaemr-337.md (hunk 5 above)
Comment (solemabrothers) [reply release-notes/ugandaemr-337.md:23]: @slubwama for your attention

File: release-notes/ugandaemr-337.md (hunk 4 above)
Comment (solemabrothers) [reply release-notes/ugandaemr-337.md]: @slubwama for your attention

File: release-notes/ugandaemr-337.md (hunk 6 above)
Comment (Mwanje) [reply release-notes/ugandaemr-337.md]: @ssmusoke more details here https://app.asana.com/0/1133167201254915/1201690883049401

File: release-notes/ugandaemr-337.md (hunk 4 above)
Comment (Mwanje) [reply release-notes/ugandaemr-337.md]: @solemabrothers we can remove it not for users

File: release-notes/ugandaemr-337.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
+# 3.3.7 - March 2022 
+
+## New Features
+
+1. Martnernal and Child Health
+  
+   * Antenatal  Services for both Point of Care and Restrospective approach. 
+   * Maternity Services
+   * Postnatal Services
+   * Reports - (105 MCH Section, ANC 105, Marternity 105, 105 PNC )
```
Comment (NaseefMayanja) [inline release-notes/ugandaemr-337.md:10]: Change to 
(105 MCH Sections for ANC, Maternity and PNC)

File: release-notes/ugandaemr-337.md (hunk 4 above)
Comment (NaseefMayanja) [reply release-notes/ugandaemr-337.md]: Change to 1. Added display board for logs

I think it's the one at Home, system administration, Ugandaemrsync, synctasklogs. If its the one, more explanation will be needed to understand the codes / wording on that board

Comment (NaseefMayanja) [reply release-notes/ugandaemr-337.md]: VL, ART Access can be monitored, if possible, add recency. I think all these have timestamps of successful data sync in the maintenance section, but as we have seen over time while troubleshooting recency, the timestamp will report the successful sync by the system but will not inform the users that the payload was blank or got an error. I think this board should be used to address that.

File: release-notes/ugandaemr-337.md (hunk 6 above)
Comment (NaseefMayanja) [reply release-notes/ugandaemr-337.md]: 6. Improved VL non suppressed form validation rules for the date when results are given to the client and the date for IAC session.

File: release-notes/ugandaemr-337.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+## Improvements
+ 
+ 1. Add Display Board for Logs
+ 2. Add SNOMED CT Mapping to Fhir Concept Source Table
+ 3. Remove dependency on guava that is flagged as a potential security issue
+ 4. Checking of results in order to also check for results entered from the encounter
+ 5. Ensure that orders with Viral Load results are completed.
+ 6. Improve VL Non Suppressed form Validation Rules
+ 7. Update Transfer-In Regimen to match the list of regimens in current regimen
+ 8. Differentiate between a pharmacy visit and a facility visit in ART Access on Patient Dashboard
```
Comment (NaseefMayanja) [inline release-notes/ugandaemr-337.md:26]: 8. Differentiate between a pharmacy visit (ART Access) and a facility visit on the patient dashboard

File: release-notes/ugandaemr-337.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+ 
+
+## Bug Fixes 
+
+1. Fix failing retrospective of Ordering Viral Load
```
Comment (NaseefMayanja) [inline release-notes/ugandaemr-337.md]: 1. Fixed failing retrospective of ordering Viral Load 

We need to be consistent, we either us VL or Viral Load throughout the release notes

File: release-notes/ugandaemr-337.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+2. Fix appointments widget with some appointments not showing.
```
Comment (NaseefMayanja) [inline release-notes/ugandaemr-337.md]: 2. Fixed the appointments widget with some appointments not showing

The tense also has to be looked at

File: release-notes/ugandaemr-337.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+**For existing versions of UgandaEMR 2.0.x to 3.3.7**
+
+
+Upgrade to 3.3.0 using the following scripts. 
+
+* 32-bit upgrade files- [https://sourceforge.net/projects/ugandaemr/files/3.3.0/ugandaemr\_upgrade\_from\_2.0.x\_to\_3.3.0\_32bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.3.0/ugandaemr_upgrade_from_2.0.x_to_3.3.0_32bit.exe/download)
+* 64-bit upgrade files- [https://sourceforge.net/projects/ugandaemr/files/3.3.0/ugandaemr\_upgrade\_from\_2.0.x\_to\_3.3.0\_64bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.3.0/ugandaemr_upgrade_from_2.0.x_to_3.3.0_64bit.exe/download)
+
+Then upload the war file for 3.3.7.
```
Comment (NaseefMayanja) [inline release-notes/ugandaemr-337.md:54]: @slubwama after this is done, do ART Access implementing facilities need to run the reset script again, or this is now embedded within the upgrade?

File: release-notes/ugandaemr-337.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+
+    *Retrospective Data Entry for SMC Services
+    
+3. Create a generic approach to generating  and Sync FHIR Encounters for data exchange
+4. Added Missing Districts
+    
+## Improvements
+ 
+ 1. Add Display Board for Logs
+ 2. Add SNOMED CT Mapping to Fhir Concept Source Table
```
Comment (NaseefMayanja) [inline release-notes/ugandaemr-337.md]: 2. Added SNOMED CT mapping to FHIR concept source table

I don't see how this is an improvement in the system. Are you trying to say that UgEMR  will communicate better with FHIR so there will be an improvement in analytics

**Pull request merged**